### ISSUE TITLE
In event profile, change "Co-author graph" to "Attendee co-author graph"

### ISSUE THREAD

**Issue by Daniel-Mietchen** [WDscholia/scholia#2060]
as suggested by @jhpoelen .

Example: https://scholia.toolforge.org/event/Q56579271#co-authors

**Comment by fnielsen** [comment]
Authors are not necessarily attendees.

**Comment by Daniel-Mietchen** [comment]
Yes, so I went for "Co-author graph of event participants" instead.

**Comment by jhpoelen** [comment]
Thanks for considering my suggestion. I agree that co-authors are not necessarily event attendees.

**Comment by fnielsen** [comment]
Wouldn't an author that did not attend the event be an "event participants"?

**Comment by fnielsen** [comment]
What about "Co-author graph of people associated with the event" (somewhat long)

**Comment by fnielsen** [comment]
My concern is that if you are in the program committee would you then be regarded as a participant? There is the particular participant property https://www.wikidata.org/wiki/Property:P710 and our query is more general than that.
